### PR TITLE
Support xclim 0.29

### LINKED ISSUE #193: Upgrade to xclim 0.29

### PR DESCRIPTION
**Pull request by huard**
## Overview

This PR fixes #193

Changes:

* Pin xclim >= 0.29
* Added boolean kind
* Added pr_25, tas_25 to recognized variables
* Support new sdba training API
* Fixed the output name creation mechanism in subset functions. The fix made in the last finch version to support PyWPS 4.4.5 could cause unnecessary downloads. 

## Additional Information

Links to other issues or sources.

### CODE REVIEW

File: tests/test_wps_xclim_indices.py [hunk 1]
```diff
@@ -44,6 +44,8 @@ def test_indicators_processes_discovery(indicator):
         parameters.remove("indexer")
         parameters.add("month")
         parameters.add("season")
+    if 'bootstrap' in parameters:
+        parameters.add("bootstrap")
```
Comment (aulemahal) [inline tests/test_wps_xclim_indices.py:48]: What is happening here? If it is in the set, add it to the set?

**Review by aulemahal — APPROVED** [review]
Seems good. Only a small question.

File: tests/test_wps_xclim_indices.py (hunk 1 above)
Comment (huard) [reply tests/test_wps_xclim_indices.py:48]: Brain malfunction.

**Comment by aulemahal** [comment]
Superseeded by #209.

**Pull request closed without merging**